grails-app/views/occurrence/_charts.gsp

```diff
@@ -3,7 +3,7 @@
 <asset:script type="text/javascript">
     <charts:biocache
             biocacheServiceUrl="${alatag.getBiocacheAjaxUrl()}"
-            biocacheWebappUrl="${grailsApplication.config.serverName}${request.contextPath}"
+            biocacheWebappUrl="${g.createLink(absolute:true, uri:'')}"
```

Comment:
I'm not sure this will work when running with an `contextPath` included. Quite a few LA sites use this.

Reply:
Hmm... AFAIK `serverName` contains the `contextPath`, so this actually fixes a bug (FWIW createLink uri will definitely handle adding the grails configured server name and context path).

But after checking, `serverName` is a non grails attribute, it may at one point have been used for CAS config but hasn't for a while, so this fixes it to use the proper grails config.

Reply:
I'm also pretty sure this works with context paths because I run all my dev systems behind a reverse proxy and with context paths

Reply:
That's fine - just wanted to check.